src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawPanelAdapter.java

```diff
@@ -5,13 +5,14 @@
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
+
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawPanelAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect.

Inheritance is not used (no super drawLine calls). Should be removed!